ush/regrid_gsiSfcIncr_to_tile.sh

```diff
@@ -102,17 +103,58 @@ for imem in $(seq 1 "${NMEM_REGRID}"); do
         MEMDIR=${memchar} YMD=${PDY} HH=${cyc} declare_from_tmpl \
             COMIN_SOIL_ANALYSIS_MEM:COM_ATMOS_ANALYSIS_TMPL
     fi
- 
+    
+    HIDE_LIAU="!"
+    rm -f "regrid.nml"
+    atparse < "${regrid_nml_tmpl}" >> "regrid.nml"
+
     for FHR in "${soilinc_fhrs[@]}"; do
         ${NCP} "${COMIN_SOIL_ANALYSIS_MEM}/${APREFIX_ENS}sfci00${FHR}.nc" \
-               "${DATA}/enkfgdas.sfci.nc"
+               "${DATA}/${in_fname}"
 
         ${APRUN_REGRID} "${REGRID_EXEC}" "${REDOUT}${PGMOUT}" "${REDERR}${PGMERR}"
 
         for n in $(seq 1 "${ntiles}"); do
-            cpfs "${DATA}/sfci.tile${n}.nc"  "${COMOUT_ATMOS_ANALYSIS_MEM}/sfci00${FHR}.tile${n}.nc" 
+            cpfs "${DATA}/${out_fname}.tile${n}.nc"  "${COMOUT_ATMOS_ANALYSIS_MEM}/${out_fname}00${FHR}.tile${n}.nc"
         done
     done
+
+    if [[ ${DO_LAND_IAU} = ".true." ]]; then 
+        
+        HIDE_LIAU=" "
+        rm -f "regrid.nml"
+        atparse < "${regrid_nml_tmpl}" >> "regrid.nml"
+        
+	      #fix until reg code time dim issues are sorted out. TODO: time dim in regr/ufs code
+        if [[ "${n_tims}" -eq 1 ]]; then
+            for FHI in "${ifhrsi[@]}"; do
+                ${NCP} "${COMIN_SOIL_ANALYSIS_MEM}/${APREFIX_ENS}sfci0${FHI}.nc" \
```

Comment:
```suggestion
                cpreq "${COMIN_SOIL_ANALYSIS_MEM}/${APREFIX_ENS}sfci0${FHI}.nc" \
```
Using `cpreq` as this is a required file